### PR TITLE
Remove some redundant consts

### PR DESCRIPTION
**Pull request by yuexiao-wang**
**- What I did**
1. Remove some redundant consts
2. Remove a dot in the option

**- How I did it**

**- How to verify it**

**- Description for the changelog**

Signed-off-by: yuexiao-wang <wang.yuexiao@zte.com.cn>

### CODE REVIEW

**Comment by duglin** [comment]
LGTM

File: cli/command/swarm/opts.go [hunk 1]
```diff
@@ -33,7 +33,7 @@ type swarmOptions struct {
 	externalCA          ExternalCAOption
 }
 
-// NodeAddrOption is a pflag.Value for listen and remote addresses
+// NodeAddrOption is a pflag.Value for listening remote addresses
```
Comment (aaronlehmann) [inline cli/command/swarm/opts.go]: Remove "remote", because this is actually a local address.

**Comment by yuexiao-wang** [comment]
@aaronlehmann PTAL
Updated

**Comment by aaronlehmann** [comment]
LGTM

**Pull request merged**